rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -126,7 +191,20 @@
                 </hint>
```

Comment:
Shouldnt it be the other way around with the quickfix ?

Reply:
Done, thanks!